### ISSUE TITLE
__version__ is missing in __init__.py

### ISSUE THREAD

**Issue by KimSangYeon-DGU** [mlpack/mlpack#2091]
#### Issue description
In python,`__version__` is missing in the `__init__.py` file. I think it would be nice if we can check the version of mlpack in python.

Possible solution:
```
// __init__.py
__version__ = "3.2.1"
```

#### Your environment

 * version of mlpack: 3.2.1
 * operating system: Ubuntu 16.04 LTS

#### Steps to reproduce
```
import mlpack
mlpack.__version__
```

#### Expected behavior
3.2.1 (or a current version of mlpack)

**Comment by piyush26c** [comment]
Tyro for this organization, working on this good first issue @KimSangYeon-DGU

**Comment by rcurtin** [comment]
@piyush26c just FYI, @KimSangYeon-DGU opened a PR in #2092, maybe you could take a look at that and see if it makes sense, perhaps help review it too?  Up to you. :+1:

**Comment by piyush26c** [comment]
Okay sir.... will look into it :+1:

**Comment by rcurtin** [comment]
Fixed via #2092. :+1:
